tests/phpunit/tests/functions/cleanDirsizeCache.php

```diff
@@ -0,0 +1,124 @@
+<?php
```

Comment:
Thanks for the PR!

There are already some tests for this function in [tests/multisite/cleanDirsizeCache.php](https://core.trac.wordpress.org/browser/tags/5.8.1/tests/phpunit/tests/multisite/cleanDirsizeCache.php), could we combine these files to avoid duplication?

I think it's only ever called in multisite context and should probably be moved from `functions.php` to `ms-functions.php`, but that might be something for another PR :)